metrics/firebase/test/firestore/rules/projects-test.js

```diff
@@ -29,37 +50,118 @@ describe("Projects collection rules", async function () {
   });
 
   it("does not allow to create a project without a name", async () => {
-    await assertFails(authenticatedApp.collection("projects").add({}));
+    await assertFails(passwordProviderAllowedDomainApp.collection("projects").add({}));
   });
 
   /**
-   * The authenticated user specific tests
+   * The google auth provider user specific tests
```

Comment:
Please rearrange tests in this file as well using the order provided in the project groups comment.